exmg-form-styles.ts

```diff
@@ -0,0 +1,64 @@
+import {html} from 'lit-element';
+import {sharedButtonStyles} from '@exmg/exmg-cms-styles/exmg-cms-button-styles.js';
```

Comment:
I think we should not use this cms-styles . I think since lit and new css features we could better redo this first.

Reply:
lets do it after related package will be updated